manticore/core/smtlib/solver.py

```diff
@@ -359,13 +364,16 @@ def can_be_true(self, constraints, expression):
 
     # get-all-values min max minmax
     #@memoized
```

Comment:
Can we remove this line? Seems we are not going to memoize it, at least for now ( ? ).